src/map/js/components/LayerPanel/LayerPanel.js

```diff
@@ -151,11 +151,13 @@ export default class LayerPanel extends React.Component {
           {landUseLayers[2]}
```

Comment:
Let's do this in a mapping function to make it more dynamic, or leave a comment in the config file to let future people know to add the layers here.